web/client/components/map/plugins/leaflet.js

```diff
@@ -9,10 +9,10 @@
 export default () => {
     require('../leaflet/plugins/index');
     return {
-        Map: require('../leaflet/Map'),
+        Map: require('../leaflet/Map').default,
```

Comment:
here in this file there is still a 
require('../leaflet/plugins/index');

is that correct?

Reply:
Yes, I didn't changed it yet